app/models/language.rb

```diff
@@ -2,44 +2,12 @@
 #
 # Table name: languages
 #
+#  created_at :datetime
 #  id         :integer          not null, primary key
+#  locale     :string
 #  name       :string
-#  code       :string
-#  created_at :datetime
 #  updated_at :datetime
 #
 
 class Language < ActiveRecord::Base
```

Comment:
That's a good option, I'd considered it but wasn't sure how well it would actually work or whether there was a possibility there was other metadata we'd want/need to store about a language, so I'd decided to leave the model. Can switch that over though and delete the model entirely. Always happy to detonate a little more code.